From a88b523e2f9709fc0ca870f8ebec42629e4768d0 Mon Sep 17 00:00:00 2001
From: zsomborklara <zsombor.klara@cloudera.com>
Date: Tue, 18 Jul 2017 16:22:19 +0200
Subject: [PATCH 1197/1363] Revert "CLOUDERA-BUILD: CDH-54238: Investigate the
 q tests which are failing on S3"        
 hive.stats.dbclass is explicitly set to fs in the
 init script. It should not be changed.

This reverts commit a20c1afc1436085e97379e1edef189fa308cc15e.

Change-Id: I3c03f0b573317200f9751c51afa794f7be35111a
---
 .../test/resources/core-blobstore/hive-site.xml    |    6 ------
 .../test/resources/hdfs-blobstore/hive-site.xml    |    6 ------
 .../test/resources/testconfiguration.properties    |   13 -------------
 3 files changed, 25 deletions(-)

diff --git a/itests/hive-blobstore/src/test/resources/core-blobstore/hive-site.xml b/itests/hive-blobstore/src/test/resources/core-blobstore/hive-site.xml
index 4c57594..038db0d 100644
--- a/itests/hive-blobstore/src/test/resources/core-blobstore/hive-site.xml
+++ b/itests/hive-blobstore/src/test/resources/core-blobstore/hive-site.xml
@@ -83,12 +83,6 @@
   </property>
 
   <property>
-    <name>hive.stats.dbclass</name>
-    <value>jdbc:derby</value>
-    <description>The storage for temporary stats generated by tasks. Currently, jdbc, hbase and counter types are supported</description>
-  </property>
-
-  <property>
     <name>javax.jdo.option.ConnectionDriverName</name>
     <value>org.apache.derby.jdbc.EmbeddedDriver</value>
   </property>
diff --git a/itests/hive-blobstore/src/test/resources/hdfs-blobstore/hive-site.xml b/itests/hive-blobstore/src/test/resources/hdfs-blobstore/hive-site.xml
index 970a976..46ae3dc 100644
--- a/itests/hive-blobstore/src/test/resources/hdfs-blobstore/hive-site.xml
+++ b/itests/hive-blobstore/src/test/resources/hdfs-blobstore/hive-site.xml
@@ -88,12 +88,6 @@
   </property>
 
   <property>
-    <name>hive.stats.dbclass</name>
-    <value>jdbc:derby</value>
-    <description>The storage for temporary stats generated by tasks. Currently, jdbc, hbase and counter types are supported</description>
-  </property>
-
-  <property>
     <name>javax.jdo.option.ConnectionDriverName</name>
     <value>org.apache.derby.jdbc.EmbeddedDriver</value>
   </property>
diff --git a/itests/src/test/resources/testconfiguration.properties b/itests/src/test/resources/testconfiguration.properties
index 83ae091..d362b91 100644
--- a/itests/src/test/resources/testconfiguration.properties
+++ b/itests/src/test/resources/testconfiguration.properties
@@ -1067,7 +1067,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   auto_join9.q,\
   auto_join_filters.q,\
   auto_join_nulls.q,\
-  auto_join_reordering_values.q,\
   auto_smb_mapjoin_14.q,\
   auto_sortmerge_join_1.q,\
   auto_sortmerge_join_10.q,\
@@ -1159,7 +1158,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   cross_join.q,\
   database.q,\
   database_location.q,\
-  database_properties.q,\
   date_join1.q,\
   dbtxnmgr_compact1.q,\
   dbtxnmgr_compact2.q,\
@@ -1180,7 +1178,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   dynpart_merge.q,\
   explain_ddl.q,\
   fileformat_sequencefile.q,\
-  filter_join_breaktask.q,\
   filter_join_breaktask2.q,\
   groupby12.q,\
   groupby1_limit.q,\
@@ -1209,7 +1206,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   groupby_multi_insert_common_distinct.q,\
   groupby_position.q,\
   groupby_sort_1.q,\
-  groupby_sort_skew_1.q,\
   groupby_sort_2.q,\
   groupby_sort_3.q,\
   groupby_sort_skew_1.q,\
@@ -1273,9 +1269,7 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   join3.q,\
   join31.q,\
   join32.q,\
-  join32_lessSize.q,\
   join34.q,\
-  join35.q,\
   join38.q,\
   join39.q,\
   join4.q,\
@@ -1309,7 +1303,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   mergejoin.q,\
   mergejoins.q,\
   mergejoins_mixed.q,\
-  metadataonly1.q,\
   multiMapJoin1.q,\
   multiMapJoin2.q,\
   multi_insert_gby.q,\
@@ -1320,7 +1313,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   multi_insert_union_src.q,\
   multi_insert_with_join.q,\
   multi_join_union.q,\
-  optimize_nullscan.q,\
   optional_outer.q,\
   orc_create.q,\
   orc_createas1.q,\
@@ -1374,7 +1366,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   parquet_write_correct_definition_levels.q,\
   partInit.q,\
   partcols1.q,\
-  part_inherit_tbl_props_empty.q,\
   partition_boolexpr.q,\
   partition_char.q,\
   partition_date.q,\
@@ -1407,7 +1398,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   partition_wise_fileformat8.q,\
   partition_wise_fileformat9.q,\
   partitions_json.q,\
-  ppd_join_filter.q,\
   protectmode.q,\
   protectmode2.q,\
   rcfile_bigdata.q,\
@@ -1517,9 +1507,7 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   union16.q,\
   union17.q,\
   union2.q,\
-  union22.q,\
   union23.q,\
-  union24.q,\
   union25.q,\
   union26.q,\
   union3.q,\
@@ -1537,7 +1525,6 @@ hdfs.blobstore.query.files=add_part_exist.q,\
   union_null.q,\
   union_remove_1.q,\
   union_remove_10.q,\
-  union_remove_15.q\
   union_remove_2.q,\
   union_remove_20.q,\
   union_remove_3.q,\
-- 
1.7.9.5

